pkg/bigquery/macros.go

```diff
@@ -21,6 +21,10 @@ func macroTimeGroup(query *sqlds.Query, args []string) (string, error) {
 	if len(args) < 2 {
 		return "", fmt.Errorf("%w: expected 2 arguments, received %d", errors.New("macro $__timeGroup needs time column and interval"), len(args))
 	}
+	
+	if(args[0] == "" || args[1] == "") {
+		return "", fmt.Errorf("macro $__timeGroup expects column name and interval but got %s and %s instead", args[0], args[1])
```

Comment:
Am I crazy or will this show empty strings like "macro $__timeGroup expects column name and interval but got and instead"? I may be mis-reading the code but that does seem like it might be a bit of an odd error message 😂

Reply:
ahh you are right, that's just dumb 😄

Reply:
Changed the code a little bit so the error message is more meaningful